0.18.1 release (#508)
packit/ogr: CHANGELOG.md

# 0.18.1

## Minor

- Added and polished `__str__` and `__repr__` methods.

## Fixes

> - Parsing bug of gitlabs URL's with subnamespace was fixed and parsing was improved.

Reviewer: ```suggestion
- A bug in parsing GitLab URLs with sub-namespace was fixed and parsing was improved.
```
